pkg/systemd/reporting.jsx

```diff
@@ -330,7 +330,7 @@ class BusWorkflowRow extends React.Component {
                     const message = cockpit.format("reportd task could not be created: $0", (exception.problem || exception.message));
 
                     this.setState({
-                        message: message,
+                        message,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)